README.md

```diff
@@ -57,6 +57,10 @@ As a contributor, you might also need the following commands:
 
 - `tldr --render <path>` render a local page for testing purposes
 
+To view tldr pages for a different language, set an environment variable `LANG` containing a valid [BCP-47](https://tools.ietf.org/html/bcp47) code. And then run the above commands as usual. Currently, we have translations for `zh`, `pt-BR`, `it`, `ta`, `fr`. For an updated list of translations, please refer to the main [tldr](https://github.com/tldr-pages/tldr) repo.
```

Comment:
Instead of phrasing this as a list of existing translations, it would be easier to just give some examples of codes that can be used. That way there won't be a need to constantly update this list as translations in more languages are contributed. For example:

```suggestion
To view tldr pages for a different language, set an environment variable `LANG` containing a valid [BCP-47](https://tools.ietf.org/html/bcp47) code (such as `zh`, `pt-BR`, or `fr`) and then run the above commands as usual. For the list of available translations, please refer to the main [tldr](https://github.com/tldr-pages/tldr) repo.
```

Reply:
Sure